source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,520 @@
 
-   .. code-block:: console
+  .. code-block:: console
 
-      vmkfstools -i clear-vmware.vmdk -d zeroedthick clear-vmware-fix.vmdk
+    $ gunzip clear-<version>-installer.iso.xz
 
-#. Add the converted image to the guest using the VMware vSphere virtual
-   machine settings.
+.. _uncompress-image-for-vmware-windows:
 
-#. Start the virtual machine.
+* For Windows:
+
+  Use `7zip`_ to uncompress it.
 
-.. _latest: https://download.clearlinux.org/image/
 .. _7zip: http://www.7-zip.org/
-.. _VirtualBox: https://www.virtualbox.org/
-.. _Intel® Virtualization Technology:
-   http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html
-.. _Intel® Virtualization Technology for Directed I/O:
-   https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices
 
+.. _method-1-vmware-esxi:
+
+Method 1: Install Clear Linux into a new VM 
+===========================================
+
+The general process for performing a fresh install of |CL| into a new VM is 
+as follows (with expanded details below):
+
+* Upload the |CL| installer ISO to the VMware server
+* Create a new VM and configure its settings
+* Attach the installer ISO to it
+* Install |CL|
+* Detach the installer ISO
+* Enable UEFI boot support
+* Power on the VM
+
+Upload the Clear Linux installer ISO to the VMware server
+*********************************************************
+
+#.  Connect to the VMware server and log into an account with root privilege.
+#.  Under the :guilabel:`Navigator` window, go to :guilabel:`Storage`.
+#.  Click :guilabel:`Datastore browser`.
+    
+    .. figure:: figures/vmware-esxi/vmware-esxi-1.png
+      :scale: 100 %
+      :alt: VMware ESXi - Navigator > Storage 
+
+    Figure 1: VMware ESXi - Navigator > Storage 
+
+#.  Click :guilabel:`Create directory` and name it `ISOs`.
+
+    .. figure:: figures/vmware-esxi/vmware-esxi-2.png
+      :scale: 100 %
+      :alt: VMware ESXi - Datastore > Create directory 
+
+    Figure 2: VMware ESXi - Datastore > Create directory 
+   
+#.  Select the newly created directory and click :guilabel:`Upload`.
+
+    .. figure:: figures/vmware-esxi/vmware-esxi-3.png
+      :scale: 100 %
+      :alt: VMware ESXi - Datastore > Upload ISO 
+
+    Figure 3: VMware ESXi - Datastore > Upload ISO 
+   
+#.  Select the |CL| installer ISO file (i.e. `clear-<version>-installer.iso`) 
+    and upload it.
+
+Create a new VM and configure its settings
+******************************************
+
+#.  Under the :guilabel:`Navigator` window, go to :guilabel:`Virtual Machines`.
+#.  On the right window, click :guilabel:`Create / Register VM`.
+
+    .. figure:: figures/vmware-esxi/vmware-esxi-4.png
+      :scale: 100 %
+      :alt: VMware ESXi - Navigator > Virtual Machines
+
+    Figure 4: VMware ESXi - Navigator > Virtual Machines
+   
+#.  :guilabel:`Select creation type`:
```

Comment:
This step lacks any verb! The steps listed as children are not numbered and they should not be children of this step. This is done multiple times across the tutorial and is always incorrect. Each step should be an action and all actions must be numbered steps. Please fix all instances.